### PR TITLE
Send emails upon submission, approval, and denial of resources

### LINKED ISSUE #47: Submit a Resource Default emails
The email copy can be found here: https://docs.google.com/document/d/1RBJyC8F0yejv6qdx3qtXwDxEK1_4P4JI_ufHrRCivbU/edit

Note that we decided we would like an auto-email for "your resource has been declined", which means that we will have to add a "declined" status under Admin.

### PR DESCRIPTION
**Pull request by rerb**
Closes #47.

### CODE REVIEW

File: hub/apps/content/templatetags/content_tags.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag
+def get_verbose_name(object):
+    return object._meta.verbose_name
```
Comment (bartTC) [inline hub/apps/content/templatetags/content_tags.py]: There is https://github.com/AASHE/hub/blob/emails/hub/apps/content/models.py#L128-L142 for this purpose.

**Comment by rerb** [comment]
@jamstooks - added tests.  :-)

**Comment by jamstooks** [comment]
Hey, Bob. Would you mind updating from master to fix the conflicts and then I'll pull this in...?

I'm about to pull in another PR, so there might be a bit more work to get it up to speed.

**Comment by rerb** [comment]
@jamstooks - Merged master down into emails branch -- no conflicts with master now.

**Comment by jamstooks** [comment]
Awesome. thanks! I've added one comment, but other than that, I think this is probably good to go...

**Pull request merged**